.github/actions/setup-and-build/action.yml

```diff
@@ -66,6 +66,7 @@ runs:
         # Check for unstaged changes
         if ! git diff --quiet; then
           echo "There are unstaged changes after build!"
+          git diff
```

Comment:
I want to keep this here - showing `git diff` with unstaged changes helps with debugging.